<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilioteApuri</class>
 <widget class="QWidget" name="TilioteApuri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="TiliCombo" name="tiliCombo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KpDateEdit" name="alkuDate"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KpDateEdit" name="loppuDate"/>
     </item>
     <item>
      <widget class="QPushButton" name="harmaaNappi">
       <property name="whatsThis">
        <string>Näytä tilille jo kirjatut tapahtumat harmaalla</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/vientilista.png</normaloff>:/pic/vientilista.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TilioteView" name="oteView">
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tositeNappi">
       <property name="text">
        <string>&amp;Tosite...</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/tekstisivu.png</normaloff>:/pic/tekstisivu.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="lisaaRiviNappi">
       <property name="text">
        <string>&amp;Kirjaa tilitapahtumia...</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/lisaa.png</normaloff>:/pic/lisaa.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="muokkaaNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Muokkaa..</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/muokkaa.png</normaloff>:/pic/muokkaa.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lisaaTyhjaBtn">
       <property name="text">
        <string>Lisää t&amp;yhjä rivi</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/lisaarivi.png</normaloff>:/pic/lisaarivi.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Poista rivi</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/poistarivi.png</normaloff>:/pic/poistarivi.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KpDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>TiliCombo</class>
   <extends>QComboBox</extends>
   <header location="global">tools/tilicombo.h</header>
  </customwidget>
  <customwidget>
   <class>TilioteView</class>
   <extends>QTableView</extends>
   <header location="global">apuri/tiliote/tilioteview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
